ldes-server-infra-mongo/mongo-repository/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/infra/mongo/dcatdataset/service/DcatDatasetEntityConverter.java

```diff
@@ -0,0 +1,27 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.dcatdataset.service;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.dcatdataset.entities.DcatDataset;
+import be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.dcatdataset.entity.DcatDatasetEntity;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.riot.Lang;
+import org.apache.jena.riot.RDFParserBuilder;
+import org.apache.jena.riot.RDFWriter;
+
+public class DcatDatasetEntityConverter {
+
+	public DcatDataset entitytoDataset(DcatDatasetEntity entity) {
+		return new DcatDataset(entity.getId(), stringToModel(entity.getModel()));
+	}
+
+	public DcatDatasetEntity datasetToEntity(DcatDataset dataset) {
+		return new DcatDatasetEntity(dataset.id(), modelToString(dataset.model()));
+	}
+
+	public String modelToString(Model model) {
+		return RDFWriter.source(model).lang(Lang.TURTLE).asString();
+	}
```

Comment:
Personally, I would make this private and try to avoid using this in other classes (in this case the test class), as this would be more consistent with the other converters in the mongo-module